res/xml/settings.xml

```diff
@@ -26,6 +26,7 @@
     <CheckBoxPreference android:key="vibrate_custom" android:title="@string/pref_vibrate_custom" android:defaultValue="false"/>
     <juloo.keyboard2.prefs.IntSlideBarPreference android:dependency="vibrate_custom" android:key="vibrate_duration" android:title="@string/pref_vibrate_duration_title" android:summary="%sms" android:defaultValue="20" min="0" max="100"/>
     <CheckBoxPreference android:key="pin_entry_enabled" android:title="@string/pref_pin_entry_enabled_title" android:summary="@string/pref_pin_entry_enabled_summary" android:defaultValue="true"/>
+    <CheckBoxPreference android:key="number_entry_enabled" android:title="@string/pref_number_entry_enabled_title" android:summary="@string/pref_number_entry_enabled_summary" android:defaultValue="true"/>
```

Comment:
Having two checkboxes for encoding three possible states feels weird. Can you change them into a `ListPreference` ?
Also, the titles of the options are confusing. The list pref could be something like "Layout when typing numbers, dates and phone numbers", "PIN entry", "number pane", "Use the main layout".
This also makes room for a "Custom" case.

Reply:
Good sugestion. Updated the code.